lib/src/bluez_client.dart

```diff
@@ -315,9 +315,14 @@ class BlueZDevice {
   int get txPower => _object.getInt16Property(_deviceInterfaceName, 'TxPower');
 
   /// UUIDs that indicate the available remote services.
-  List<BlueZUUID> get uuids => _object
-      .getStringArrayProperty(_deviceInterfaceName, 'UUIDS')
-      .map((value) => BlueZUUID(value));
+  List<String> get uuids {
+    var value = _object.getStringArrayProperty(_deviceInterfaceName, 'UUIDs');
```

Comment:
This intentionally returns `BlueZUUID` (see #2), please re-submit with just the typo fix.